examples/subscription_handler_example.py

```
@@ -0,0 +1,668 @@
+
+
+@subscription.source("counter")
+async def counter_generator(*_, limit: int = 5):
+    """Simple counter that emits integers."""
+    for i in range(limit):
+        yield i
+        await asyncio.sleep(1)
+
+
+@subscription.field("counter")
+def resolve_counter(count, *_, **kwargs):
```

Ruff (0.15.1)

[warning] Unused function argument: `kwargs`

(ARG001)